enzy_htp/mutation/engine.py

```diff
@@ -226,6 +230,72 @@ def _mutate_pymol(pdb: str, outfile: str, mutations: List[Mutation]) -> None:
     pass
 
 
+def _mutate_rosetta(pdb:str, outfile:str, mutations: List[Mutation]) -> None:
```

Comment:
This file is deprocatced. Maybe find a way to move this function to api.py and mimic the format of mutate_stru_with_tleap().

(May be do this after merging is easier?)

Reply:
Disregard this function for now. I will rewrite it and mimic what is being done for pymol/amber.

Reply:
cool